File: SampleAppObjC/sources/simple/NoStoryboardSampleVC.m
//
//  NoStoryboardSampleVC.m
//  SampleAppObjC
//
//  Copyright © 2017-2018 BEAR SAS. All rights reserved.
//

#import "NoStoryboardSampleVC.h"
#import "NoStoryboardViewController.h"

@interface NoStoryboardSampleVC ()

@end

@implementation NoStoryboardSampleVC

- (IBAction)showBearScanner {
    NoStoryboardViewController* vc = [[NoStoryboardViewController alloc] init];
    vc.modalPresentationStyle = UIModalPresentationFullScreen;
    [vc setDelegate:vc];
    [vc setScannerColor:[UIColor blackColor]];
    [vc setTimeToPause:5];
    
    [self showViewController:vc sender:self];
}

- (IBAction)goBack {
    [self dismissViewControllerAnimated:YES completion:nil];
}

@end

File: SampleApp/sources/simple/NoStoryboardSampleVC.swift
//
//  NoStoryboardSampleVC.swift
//  SampleApp
//
//  Copyright © 2017 BEAR SAS. All rights reserved.
//

import UIKit
import BearSDK

final class NoStoryboardSampleVC: UIViewController {
    
    @IBAction func showBearScanner() {
        let vc = NoStoryboardViewController()
        vc.delegate = vc
        vc.scannerColor = UIColor.black
        vc.timeToPause = 5
        
        show(vc, sender: self)
    }
    
    @IBAction func goBack() {
        dismiss(animated: true, completion: nil)
    }
}
